### ISSUE TITLE
update logic for options of loss function for dpctgan and dpgan 

### ISSUE THREAD

**Issue by AprilXiaoyanLiu** [opendifferentialprivacy/whitenoise-system#273]
need update logic for options of loss functions for dpctgan and dpgan. 
possibly add a try except to enable loss can be only cross_entropy or wasserstein for the moment
